src/main/java/team9/ddang/walk/controller/WalkLocationController.java

```diff
@@ -0,0 +1,28 @@
+package team9.ddang.walk.controller;
+
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.stereotype.Controller;
+import team9.ddang.member.entity.Member;
+import team9.ddang.member.repository.MemberRepository;
+import team9.ddang.walk.controller.request.LocationRequest;
+import team9.ddang.walk.service.WalkLocationService;
+
+@Slf4j
+@Controller
+@RequiredArgsConstructor
+@Tag(name = "Walk WebSocket API", description = "산책 웹소켓 API")
+public class WalkLocationController {
+
+    private final WalkLocationService walkLocationService;
+    private final MemberRepository memberRepository;
+
+    @MessageMapping("/api/v1/location")
+    public void updateUserLocationTest(LocationRequest locationRequest) {
```

Comment:
이거 저도 궁금해서 찾아봤는데, Spring WebSocket에서는 기본적으로 MessageMapping 메서드에서 자동으로 유효성 검증이 수행되지 않는다고 해요. Valid 애노테이션 추가해야 DTO 유효성을 검사한다고 합니다.

또한 전역 예외 처리도 WebSocket 클라이언트에 반환하려면 WebSocket 전역 예외 처리기를 작성해야 하는데, 그건 이번 제 PR에 올라갈 예정입니다.

Spring 시큐리티가 없어서 SendTo로 모든 웹소켓 구독자를 대상으로 실험해봤는데, 잘 동작해요.
![image](https://github.com/user-attachments/assets/56fc369d-afdf-487b-a0d5-52ec7e8320eb)


```suggestion
    public void updateUserLocationTest(@Valid LocationRequest locationRequest) {
```

Reply:
아하 넵 감사합니다!